source/bot/index.ts

```diff
@@ -1,5 +1,7 @@
 import * as process from 'node:process';
 
+import {config as dotenv} from 'dotenv';
+
 import {Bot, session} from 'grammy';
```

Comment:
Removing empty line  and sort the entries

```suggestion
import {Bot, session} from 'grammy';
import {config as dotenv} from 'dotenv';
```